.travis.yml

```diff
@@ -36,6 +36,10 @@ jobs:
     # ---------------------------------------
     - stage: lint
       name: 'Linter'
+      if: (type IN (pull_request))
+        AND (branch IN (master, 4-dev, 3-dev, 2-dev, 2-stable))
+        AND commit_message !~ /(skip-ci|wip)/
```

Comment:
Don't you should write something like
```
 name: 'Linter'
    if: "!contains(github.event.head_commit.message, 'skip-ci')"```
```
instead? @alexandrebouthinon 
https://github.com/marketplace/actions/skip-based-on-commit-message

Reply:
I used travis [conditionnal builds](https://docs.travis-ci.com/user/conditions-v1)

Reply:
You r right. I dont know why I thought it was a travis configuration file -_-

Reply:
No need to add an explicit condition, Travis handle it for you https://docs.travis-ci.com/user/customizing-the-build/#skipping-a-build

Reply:
There is a better way to do that let's talk about it on Discord